app/shell-window/ui.js

```diff
@@ -42,11 +42,14 @@ export function setup (cb) {
 }
 
 function onWindowEvent (event, type) {
-  if (type == 'blur') { document.body.classList.add('window-blurred') }
-  if (type == 'focus') {
-    document.body.classList.remove('window-blurred')
-    try { pages.getActive().webviewEl.focus() } catch (e) {}
+  switch (type) {
+    case 'blur': return docuument.body.classList.add('window-blurred')
```

Comment:
typo here

Reply:
herp. Where my linting at! I'll use the script you've made next time (assuming that's the preferred pattern at the moment)

Reply:
ooooo, that linting setup is stale D:

Reply:
Yeah I need to run it again and clean things up